packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -604,17 +604,12 @@ export function deferredUpdates<A>(fn: () => A): A {
   return runWithPriority(NormalPriority, fn);
 }
 
-export function interactiveUpdates<A, B, C, R>(
+export function discreteUpdates<A, B, C, R>(
   fn: (A, B, C) => R,
   a: A,
   b: B,
   c: C,
 ): R {
-  if (workPhase === NotWorking) {
-    // TODO: Remove this call. Instead of doing this automatically, the caller
-    // should explicitly call flushInteractiveUpdates.
-    flushPendingDiscreteUpdates();
-  }
   if (!revertPassiveEffectsChange) {
     // TODO: Remove this call for the same reason as above.
```

Comment:
Let's remove this `flushPassiveEffects` call now.

Reply:
Remove `flushPassiveEffects` altogether and the flag, as it won't be used anywhere?

Reply:
Yeah

Reply:
It's not a breaking change because this is only observable via the event system or via `unstable_interactiveUpdates`. After this PR, the event system will call `flushDiscreteUpdates`, which itself calls `flushPassiveEffects` behind a flag. So the behavior will be the same.